src/main/java/bio/terra/metadata/AssetRelationship.java

```diff
@@ -6,17 +6,20 @@
     private UUID id;
     private StudyRelationship studyRelationship;
 
+    public AssetRelationship() {}
```

Comment:
Do you need the default constructor? There is another spot above as well. It is not usually needed, as far as I know.